src/common/DataObjectRepository.cpp

```diff
@@ -2244,9 +2244,7 @@ RESQML2_NS::SealedVolumeFrameworkRepresentation* DataObjectRepository::createSea
 
 RESQML2_NS::AbstractIjkGridRepresentation* DataObjectRepository::createPartialIjkGridRepresentation(const std::string & guid, const std::string & title)
 {
-	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = gsoap_resqml2_0_1::soap_new_eml20__DataObjectReference(getGsoapContext());
-	dor->UUID = guid;
-	dor->Title = title;
+	gsoap_resqml2_0_1::eml20__DataObjectReference* dor = createDor(guid, title, "");
```

Comment:
Would It be more appropriate to choose the DataObjectReference version (EML 2.0 or EML 2.3) depending on the value of getDefaultResqmlVersion()?

Reply:
I think you mix the instance version with the schema version.
The empty string on this line is about the instance version.
The schema version is on the lines just below.

We might want to add a version parameter to the method for the instance version of the partial dataobject. I have not strong opinion about that.

Reply:
Ok. Thanks for your reply.